### ISSUE TITLE
End point for Angular config/init data

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#9403]
We need to create a rest endpoint to obtain required configuration data for the Angular client.

One example of the data to return is the URL used to connect to the Websockets, right now in the Angular core the url is hardcoded:

https://github.com/dotCMS/core-web/blob/master-3.7/src/api/services/dotcms-events-service.ts#L14
```
    connectWithSocket() {
        // TODO: how are we going to handle this url?
        this.ws = new $WebSocket("ws://localhost:8080/system/events");
        this.ws.send();
    }
```

This new resource will be the first service called by angular after a successful login is made.

**Comment by jcastro-dotcms** [comment]
PR: https://github.com/dotCMS/core/pull/9432

**Comment by jcastro-dotcms** [comment]
PR core: https://github.com/dotCMS/core/pull/9432
PR core-web: https://github.com/dotCMS/core-web/pull/154

**Comment by jcastro-dotcms** [comment]
Hi @john-thomas-dotcms, please document the following properties:

* dotcms.websocket.protocol
* dotcms.websocket.baseurl
* websocket.systemevents.endpoint

Let me know when you are working on this and I'll send you the tech doc for them.

**Comment by jcastro-dotcms** [comment]
PR related to fixing problem with non-authenticated requests: https://github.com/dotCMS/core/pull/9466

**Comment by bryanboza** [comment]
Fixed, tested on master-3.7 // Postgres // FF

**Comment by john-thomas-dotcms** [comment]
Documented (though I think there's probably more to do here before we release): http://auth.dotcms.com/docs/latest/system-events
